src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -3,65 +3,152 @@
 import io.restassured.RestAssured;
 import io.restassured.response.ExtractableResponse;
 import io.restassured.response.Response;
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
 
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+import java.util.stream.Stream;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 @DisplayName("지하철역 관련 기능")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class StationAcceptanceTest {
+
+    private final String STATION_API_URL = "/stations";
+
+    static Stream stationNames() {
+        return Stream.of("강남역");
+    }
```

Comment:
```suggestion
    static Stream stationNames() {
        return Stream.of(1);
    }
```
Generic 타입을 명시해주시는 것을 추천드립니다 
위와 같이 바꾸어도 컴파일 타임 에러가 나지 않습니다. 💦